src/sentry/static/sentry/app/components/dropdownMenu.jsx

```diff
@@ -101,7 +101,12 @@ class DropdownMenu extends React.Component {
       onClickOutside(e);
     }
 
-    this.handleClose(e);
+    // Timeout to handle clicks in react portals (e.g. `Hovercard`)
```

Comment:
I would avoid the word `timeout` since it kind of implies waiting after a certain time elapses. Here, we're just kind of ~ab~using the event loop a bit